source/rust_verify/src/rust_to_vir.rs

```diff
@@ -65,7 +66,7 @@ fn check_item<'tcx>(
                 None,
                 None,
                 generics,
-                body_id,
+                Either::Left(body_id),
```

Comment:
My understanding is that the general rust ethos is to use dedicated descriptive enums for cases like these, so that it's un-ambiguous what `Left` and `Right` represent (which is also why there's no `Either` in `std`).
It would also clarify what's the role of each branch of the `match` on `body_id` in `rust_to_vir_func`.

Reply:
I just removed the use of `Either` and replaced it with a dedicated enum for this case.